app/code/Magento/InventoryInStorePickupApi/Api/Data/PickupLocationInterface.php

```diff
@@ -3,15 +3,140 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-declare(strict_types=1);
-
 namespace Magento\InventoryInStorePickupApi\Api\Data;
 
-/**
- * @TODO Replace with autogenerated immutable realization.
- * @see Please check issue for more details: https://github.com/magento-engcom/msi/issues/2126
- */
-interface PickupLocationInterface
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+interface PickupLocationInterface extends ExtensibleDataInterface
```

Comment:
Please revert the strict types declaration

Reply:
But... [1.3.1](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html)

Reply:
Let's override 1.3.1. Sometimes rules should be broken for a greater purpose.